plugins/ssl/cert_validity

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
It would be great, if you could switch to `sh`. This would allow the plugin to be used with OpenWrt or other embedded environments.

I commented the `bash`-only features below.